main.md

```diff
@@ -29,8 +29,8 @@ We aim to create a world in which any interested party can mine Spacemesh crypto
 ### Energy consumption due to PoW chains is skyrocketing.
 Spacemesh has improved upon PoW by creating a consensus protocol that consumes significantly less energy (i.e., `A Better Bitcoin`).
 
-### Lack of satisfactory smart contracts infrastructure for creation of fair communities and decentralized financial services.
-We plan to add a modern smart contracts virtual machine, called Spacemesh virtual machine (SVM), a new modern smart contracts language (Smesh) and dev tools designed for developers to build applications on the Spacemesh platform.
+### Lack of satisfactory smart contract infrastructure for creation of fair communities and decentralized financial services.
```

Comment:
What is a "fair community"?
More generally, the explanation below says nothing about the (potential) uses of smart contrasts. I believe that we should simply write "Lack of satisfactory smart contract infrastructure". What exactly the world will find fit to do with it is not one of our goals.